rollup.config.js

```diff
@@ -63,7 +65,14 @@ const configs = ['posthog-node', 'posthog-web', 'posthog-ai'].reduce((acc, x) =>
     {
       input: `./${x}/lib/${x}/index.d.ts`,
       output: [{ file: `./${x}/lib/index.d.ts`, format: 'es' }],
-      plugins: [dts()],
+      plugins: [
+        dts(),
+        alias({
+          entries: [
+            { find: './surveys-types', replacement: path.resolve(__dirname, 'posthog-core/src/surveys-types') }
+          ]
+        })
+      ],
```

Comment:
because of this running tests break everything, so tests will follow up once I fix this but I don't think we need to delay the beta release, its under a flag anyway, requires adding the survey provider, so I don't see as a big pro.